server/src/core/symbols/module_symbol.rs

```diff
@@ -184,23 +185,27 @@ impl ModuleSymbol {
                     match key {
                         Expr::StringLiteral(key_literal) => {
                             let key_str = key_literal.value.to_string();
+                            if visited_keys.contains(&key_str){
+                                add_diagnostic(&mut res, self._create_diagnostic_for_manifest_key("A manifest should not have duplicate keys", S!("OLS30202"), &key_literal.range, Some(DiagnosticSeverity::WARNING)), &session.current_noqa);
```

Comment:
new code means you should add them to the error code list

Reply:
Oh I did it but I did not stage it while committing, sorry

Reply:
amended, I also had noticed more missing error codes and added them too